src/core/libmaven/datastructures/mzSlice.cpp

```diff
@@ -3,6 +3,31 @@
 #include "masscutofftype.h"
 #include "mzMassCalculator.h"
 
+mzSlice::mzSlice(float minMz, float maxMz, float minRt, float maxRt)
+{
+	mzmin = minMz;
+	mzmax = maxMz;
+	rtmin = minRt;
+	rtmax = maxRt;
+	mz = minMz + (maxMz - minMz) / 2;
```

Comment:
This can be reduced to `mz = (minMz + maxMz)/2` and `rt = (minRt + maxRt)/2`

Reply:
Yes it can.